department-of-veterans-affairs/caseflow
# HunJerBAH legacy seed script edits

# Description
Updated legacy appeal script to create MST/PACT legacy appeals and assign the attorney task on the appeal to a user

## Acceptance Criteria
- [ ] Legacy appeals are created when the script is run
- [ ] The attorney tasks on the appeals are assigned to the user entered. 

## Testing Plan
1. Open up the rails console in local or UAT.
2. Run `bundle exec rake 'db:generate_legacy_appeals[true]'`\
3. When the option comes to enter a user CSS id, enter one of the IDs shown on the prompt for the respective environment. The user **must** be an attorney user for the task to be interactable. If you do not need to access the decision workflow, you can select any user CSS ID.
4. When the script finishes running, navigate to Caseflow and sign in as that user you listed in the script.
5. Navigate to the user's Queue. You should see the new legacy appeals assigned to the user.
6. Click on one of the appeals to navigate to the case details page.
7. On the case details page, make sure you can click the task dropdown to navigate to the decisions workflow for that attorney user.


## Screenshots
![image](https://github.com/department-of-veterans-affairs/caseflow/assets/99915461/fb779106-12c6-4414-b79c-49874dce8a0d)
![image](https://github.com/department-of-veterans-affairs/caseflow/assets/99915461/ea6cb405-ff45-456e-899b-e53ad32f5c12)
![image](https://github.com/department-of-veterans-affairs/caseflow/assets/99915461/c652bd79-023b-434c-9143-85b36c20af58)
![image](https://github.com/department-of-veterans-affairs/caseflow/assets/99915461/14e4c59a-32a8-433f-a808-18b98c813be7)
